lib/inline_svg/action_view/helpers.rb

```diff
@@ -12,7 +12,7 @@ def inline_svg(filename, transform_params={})
             configured_asset_file.named filename
           end
         rescue InlineSvg::AssetFile::FileNotFound
-          return "<svg><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
+          return "<svg class='svg-not-found'><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
```

Comment:
My first thought was to make this configurable, but then I saw that this gem doesn't have configuration already. Maybe that might make sense to add as well? It would certainly help out the efforts of https://github.com/jamesmartin/inline_svg/issues/48